index.bs

```diff
@@ -391,6 +470,76 @@ interface mixin WebTransport {
         {{WebTransportCloseInfo/errorCode}} and its reason value set to the
         value of {{WebTransportCloseInfo/reason}}.
 
+: <dfn for="WebTransport" method>getStats()</dfn>
+:: Gathers stats for this {{WebTransport}}'s QUIC
+   connection and reports the result asynchronously.</p>
+
+   When close is called, the user agent MUST run the following steps:
+     1. Let |transport| be the WebTransport on which `getStats` is invoked.
+     1. Let |p| be a new promise.
+     1. Return |p| and continue the following steps in background.
+         1. Gather the stats from the underlying QUIC connection.
+         1. Once stats have been gathered, resolve |p| with the
+            {{WebTransportStats}} object, representing the gathered stats.
+
+## Configuration ##  {#web-transport-configuration}
+
+<pre class="idl">
+dictionary WebTransportOptions {
+  sequence&lt;RTCDtlsFingerprint&gt; serverCertificateFingerprints;
+};
+</pre>
+
+<dfn dictionary>WebTransportOptions</dfn> is a dictionary of parameters
+that determine how WebTransport connection is established and used.
+
+: <dfn for="WebTransportOptions" dict-member>serverCertificateFingerprints</dfn>
```

Comment:
I think it would be less ambiguous to use a non-normative description here and put the normative behaviour in the algorithm steps for the constructor.

Reply:
Agreed, though I'd have to write more text about how connection establishment happens in general, so perhaps that would be better left for a follow-up CL.